opentelemetry/proto/profiles/v1experimental/pprofextended.proto

```diff
@@ -354,9 +354,6 @@ message Location {
   // profile changes.
   bool is_folded = 5;
 
-  // Type of frame (e.g. kernel, native, python, hotspot, php). Index into string table.
-  int64 type_index = 6;
-
   // References to attributes in Profile.attribute_table. [optional]
   repeated uint64 attributes = 7;
```

Comment:
What's about indexes for the following fields? Shouldn't it be changed to `-1`?

Reply:
With this change, https://github.com/open-telemetry/opentelemetry-proto/pull/575 and more upcoming changes, it might be easier to update the indexing after all changes landed. That way it is less of a breaking change, until the index cleanup it done.